.gitignore

```diff
@@ -3,3 +3,6 @@ __pycache__/
 dist
 site
 data
+.venv
+.DS_Store
+.gitignore
```

Comment:
Is there a particular reason you include the gitignore in itself?

Reply:
No, just a habit, so that I don't have to take care of strange OS and editors of contributors, like you have to right now :)